force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js

```diff
@@ -376,24 +378,23 @@ export default class RecurringDonationTable extends LightningElement {
                             return action;
                         });
                     }
+                    el.recurringDonation.npe03__Next_Payment_Date__c = new Date(el.recurringDonation.npe03__Next_Payment_Date__c).toLocaleDateString(undefined, { timeZone: this.timeZone });
                     let lastModifiedDate = new Date(el.recurringDonation.LastModifiedDate).toLocaleDateString(undefined, { timeZone: this.timeZone });
                     return { actions, ...el, nexDonationFormatFirstElement, nexDonationFormatSecondElement, lastModifiedDate };
                 });
             }
         }).finally(() => {
           this.data?.forEach((item) => {
             let nextDonationHtml = `<div class="${this.rowClasses}" style="${this.fixedWidth}">`;
-            if(item.recurringDonation.npe03__Next_Payment_Date__c){
+            if(item.recurringDonation.npe03__Next_Payment_Date__c !== "Invalid Date"){
```

Comment:
Where does this string come from? Will it work even if the users locale/language is not English?